src/os/unix/unix.cpp

```diff
@@ -28,6 +28,10 @@
 #include <SDL.h>
 #endif
 
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
```

Comment:
I totally missed this, but otherwise my fellow devs will hurt me: please add a tab between `#` and `include`, as seen in the blob just below this.
Currently not all code is converted yet, but new code should fix this :)